breeze

```diff
@@ -724,14 +724,14 @@ function breeze::parse_arguments() {
             shift 2
             ;;
         -i | --integration)
-            local integration=${2}
+            local INTEGRATION=${2}
             parameters::check_and_save_allowed_param "INTEGRATION" "integration" "--integration"
```

Comment:
```suggestion
            parameters::check_and_save_allowed_param "integration" "integration" "--integration"
```

Reply:
I thought I tired this, and it then couldn't find the list of allowed values.

I will double check